lib/chat_models/chat_anthropic.ex

```diff
@@ -366,15 +366,17 @@ defmodule LangChain.ChatModels.ChatAnthropic do
         tools,
         retry_count
       ) do
+    config_req_opts = Config.get_env(:req_opt, [])
+
     req =
       Req.new(
         url: url(anthropic),
         json: for_api(anthropic, messages, tools),
         headers: headers(anthropic),
         receive_timeout: anthropic.receive_timeout,
-        retry: :transient,
-        max_retries: 3,
-        retry_delay: fn attempt -> 300 * attempt end,
+        retry: Keyword.get(config_req_opts, :retry, :transient),
+        max_retries: Keyword.get(config_req_opts, :max_retries, retry_count),
+        retry_delay: Keyword.get(config_req_opts, :retry_delay, &req_default_retry/1),
         aws_sigv4: aws_sigv4_opts(anthropic.bedrock)
       )
```

Comment:
It would be useful if this worked for any req option, not only retry config. How about this?

```suggestion
      )
      |> Req.merge(config_req_opts)
```

Reply:
Yep it makes sense.